bibliopixel/animation/animation.py

```diff
@@ -23,21 +33,44 @@ def _led(self, layout):
     def preRun(self, amt=1):
         self.layout.all_off()
 
+    @property
```

Comment:
I was doing this so I could verify it's actually a function so someone cannot set it to 3 or something. And on_completion is better. Will change.